### DISCUSSION TITLE
How to know about mentors

### DISCUSSION THREAD

**Discussion by Bharathhh30** [sanjay-kv/Open-source-Practice#560] [Q&A] [answered]
Can anyone help me out with finding mentor and project admin , i am a beginner and interested in contributing in the path of python

**Comment by MuraliDharan7** [comment]
You can know your mentor here by searching your project:

Mentor Assigned to the Project List: https://bit.ly/3UVRkgM

**Comment by TonyStark-47** [comment]
You can see it here, there names and email addresses project wise:
https://docs.google.com/spreadsheets/d/1nr5EE-Yad9MaSVaQLbiLEMJWPuDNeyhFWVHxZGvkVbA/edit

**Comment by Pranali3103** [comment, marked as answer]
You can know about  your mentors from 
Step:-01 Go to GSSOC repository on GitHub .
Link is :https://github.com/GSSoC24

Step:-02 Go to repositories inside GSSOC'24
Link is :https://github.com/orgs/GSSoC24/repositories

Step:-03 Go to Contributor's repository 
Link is :- https://github.com/GSSoC24/Contributor

Step:-04 Scroll down a bit and go on to Mentor Assigned to the Project List:
Link is :-https://bit.ly/3UVRkgM

OR DIRECTLY GO THIS LINK :
https://bit.ly/3UVRkgM

You'll find mentors assigned to task according to project you want to contribute

**Comment by rugved0102** [comment]
Here's the mentor list : https://docs.google.com/spreadsheets/d/1nr5EE-Yad9MaSVaQLbiLEMJWPuDNeyhFWVHxZGvkVbA/edit#gid=0

**Comment by Ravindrak0812** [comment]
Click This link you are directly navigate to mentor list  page-->(url:"https://docs.google.com/spreadsheets/d/1nr5EE-Yad9MaSVaQLbiLEMJWPuDNeyhFWVHxZGvkVbA/edit#gid=0")
